pkg/http/api.go

```diff
@@ -224,6 +233,29 @@ func (a *api) MarkStatusAsOutboundReady() {
 	a.outboundReadyStatus = true
 }
 
+func (a *api) constructWorkflowEndpoints() []Endpoint {
+	return []Endpoint{
+		{
+			Methods: []string{fasthttp.MethodGet},
+			Route:   "workflows/{workflowId}/{workflowRunId}",
+			Version: apiVersionV1alpha1,
+			Handler: a.onGetWorkflow,
+		},
+		{
+			Methods: []string{fasthttp.MethodPost},
+			Route:   "workflows/{workflowName}",
+			Version: apiVersionV1alpha1,
+			Handler: a.onStartWorkflow,
+		},
+		{
+			Methods: []string{fasthttp.MethodDelete},
+			Route:   "workflows/{workflowId}/{workflowRunId}",
+			Version: apiVersionV1alpha1,
+			Handler: a.onTerminateWorkflow,
+		},
```

Comment:
We'll need to update these routes to match the latest iteration of the spec. Some of the variable names are also different in the spec (I assume you're using the Temporal terminology here). Ping me for details.